samples/snippets/src/main/java/pubsublite/spark/AdminUtils.java

```diff
@@ -208,4 +220,51 @@ public static void publisherExample(
       }
     }
   }
+
+  public static List<PubsubMessage> subscriberExample(
+          String cloudRegion, char zoneId, long projectNumber, String subscriptionId)
+          throws ApiException {
+    CloseableMonitor m = new CloseableMonitor();
+    List<PubsubMessage> result = new ArrayList<>();
+
+    SubscriptionPath subscriptionPath =
+            SubscriptionPath.newBuilder()
+                    .setLocation(CloudZone.of(CloudRegion.of(cloudRegion), zoneId))
+                    .setProject(ProjectNumber.of(projectNumber))
+                    .setName(SubscriptionName.of(subscriptionId))
+                    .build();
+
+    MessageReceiver receiver =
+            (PubsubMessage message, AckReplyConsumer consumer) -> {
+              try (CloseableMonitor.Hold h = m.enter()) {
```

Comment:
You might wish to explain this a bit since the JavaDoc doesn't.  
https://googleapis.dev/java/google-cloud-pubsublite/latest/com/google/cloud/pubsublite/internal/CloseableMonitor.html
https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/util/concurrent/Monitor.html
https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/util/concurrent/Monitor.Guard.html

Reply:
Agree with @lesv. I have not seen `CloseableMonitor` used in receiver. What's the benefit here?

Reply:
It should not be here. This is not a public class

Reply:
Fixed, will make sure those won't surface to public level.